app/models/service_template.rb

```diff
@@ -69,6 +69,70 @@ class ServiceTemplate < ApplicationRecord
   scope :with_existent_service_template_catalog_id, ->         { where.not(:service_template_catalog_id => nil) }
   scope :displayed,                                 ->         { where(:display => true) }
 
+  def self.catalog_item_types
+    CATALOG_ITEM_TYPES.each.with_object({}) do |(key, description), hash|
```

Comment:
How does the UI use it - gray out non-displayable item from the dropdown? Or should we simply remove non-qualified types?

Reply:
@bzwei @h-kataria mentioned that there is some bootstrap magic that does that.

Reply:
@bzwei yes UI will show those items as grayed/disabled in the drop down, see screenshot in https://github.com/ManageIQ/manageiq-ui-classic/pull/2908